Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'okcomputer'
 gem 'pg'
 gem 'propshaft'
 gem 'puma' # app server
-gem 'rails', '~> 7.2.0'
+gem 'rails', '~> 8.0'
```

Comment:
Maybe I've missed this in other projects, but I think we want to stay pinned to the patch level (8.0.0).